packages/release/mnt.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Mnt Directory (/mnt)
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+Wants=prepare-local.service
+After=prepare-local.service
+
+[Mount]
+What=/local/mnt
+Where=/mnt
+Type=none
+Options=rbind,rshared
```

Comment:
This will change the effective default label of `/mnt` from `external_t` to `local_t`.

When you mount a newly formatted filesystem, what label does its mount point end up with?

Reply:
New mount points are labeled with the `system_u:object_r:unlabeled_t:s0` label:

```
[ec2-user@<> ~]$ ls -Z /.bottlerocket/rootfs/mnt/
drwxr-xr-x. root root system_u:object_r:unlabeled_t:s0 part1
drwxr-xr-x. root root system_u:object_r:unlabeled_t:s0 part2
```